### PR TITLE
Added gif of data explorer to package README

### PR DESCRIPTION
**Pull request by MSeal**
Added Elijah's gif to the package readme. Only concern is that this adds 1.4MB to the git history

### CODE REVIEW

**Comment by mpacer** [comment]
Lovely! LGTM! I'd merge it but I feel like @emeeks should have the privilege :)

File: packages/data-explorer/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 # nteract Data Explorer
 
+![](sampo.gif)
```
Comment (captainsafia) [inline packages/data-explorer/README.md]: Can you add a description to this image for accessibility purposes?

**Comment by BenRussert** [comment]
> Only concern is that this adds 1.4MB to the git history

If this is a concern, we usually handle these on the hydrogen repo by adding a link to a hosted image (usually from a github comment) instead of committing the image.

**Comment by MSeal** [comment]
Using this to post the image uri :D

sampo

**Comment by MSeal** [comment]
Gonna go ahead and merge to get this out of the list of available PRs since it's been up for 10 days and has +1 from folks :)

**Pull request merged**